src/main/java/io/r2dbc/postgresql/type/PostgresqlObjectId.java

```diff
@@ -399,12 +399,7 @@ public static boolean isValid(int objectId) {
 
         if (objectId >= 0 && objectId < OID_CACHE_SIZE) {
             PostgresqlObjectId oid = CACHE[objectId];
-
-            if (oid == null) {
-                return false;
-            }
-
-            return true;
+            return oid == null ? false : true;
```

Comment:
This could be simplified even more to ˋreturn oid != nullˋ.

Reply:
addressed it.